src/test/java/com/google/sps/SessionContextTest.java

```diff
@@ -32,6 +32,7 @@ public void setup() {
     @Test
     public void getLoggedInUser_ReturnUser() {
         User testUser = new User("Bob");
+        testUser.setId("444");
```

Comment:
Beyond the scope of this PR, but I'd consider changing User so it takes an ID instead of / in addition to the name on construction (since users need to have an ID to work in the system). Not at all a high priority refactor though.

Reply:
Good point